mavros_extras/src/plugins/mag_calibration_status.cpp

```diff
@@ -99,6 +99,10 @@ class MagCalStatusPlugin : public plugin::Plugin
     mavlink::common::msg::MAG_CAL_REPORT & mr,
     plugin::filter::SystemAndOk filter [[maybe_unused]])
   {
+    if (mr.compass_id >= calibration_show.size()) {
+      return ;
```

Comment:
https://github.com/mavlink/mavros/runs/4486152998?check_suite_focus=true#step:4:1899

Please fix style.

Reply:
What does it mean? Should I erase the gap between return and ;?

Reply:
I couldn't build the environment for some reason, so I couldn't directly test the build. Sorry to bother you. :(